### DISCUSSION TITLE
VS code - intellicode - Java Extension fails to activate. Error while activating Jav

### DISCUSSION THREAD

**Discussion by LarsCalco** [microsoft/vscode-discussions#1046] [Extension Development QnA] [answered]
I have an error with running scripts. I can run scripts once, then an error occurs and it no longer runs the script. See included image for elaborate error. When I press the run script button nothing happens, nothing is shown in the terminal. I found that if you switch to OUPUT and click VS IntelliCode in the dropdown menu it shows the error in the included image. 

What I have tried: 
- restarting laptop, reinstalling VS code, JDK, including deleting %appdata%
- if I delete the folder which is named in the error, it can compile + run the script once again, but then same error comes back
- changing laptop, error returns so it has something to do with a folder in the onedrive which syncs
- I installed everything locally, even changed the workspacestorage to a local folder, error still persists so it is not due to this map being in onedrive
- running as adminstrator
- file path is not too long, as I checked with installing everything directly on C:/

Any help and tips are welcome! 

Lars

 (whited out part is for privacy reasons)
![error code vs code](https://github.com/microsoft/vscode-discussions/assets/155455962/be665980-a5ff-4b29-bb5f-35188d99dc5a)

**Comment by gjsjohnmurray** [comment, marked as answer]
This forum is focused on extension developers. Please see the pinned [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) post for links to other channels more appropriate for your issue.
